hera_cal/abscal.py

```diff
@@ -3566,6 +3566,7 @@ def post_redcal_abscal_run(data_file, redcal_file, model_files, raw_auto_file=No
         echo("The following model files overlap with data files in LST:\n" + "\n".join(matched_model_files), verbose=verbose)
         hd = io.HERAData(data_file)
         hdm = io.HERAData(matched_model_files)
+        hc.gain_scale = hdm.vis_units  # set vis_units of hera_cal based on model files.
```

Comment:
I'd suggest that if this changes the units on hc, a warning is issued. This seems like the right thing to do, but I don't think it should be silent.

Reply:
Specifically, I think the warning should say what the units were and what they are getting set to.

Reply:
Should I even issue a warning if the units were 'UNCALIB'?

Reply:
I'd issue a warning in any case where the units change and the original units weren't UNCALIB

Reply:
I've added a warning.